module/zfs/zfeature_common.c

```diff
@@ -242,4 +242,8 @@ zpool_feature_init(void)
 	    "Support for blocks larger than 128KB.",
 	    ZFEATURE_FLAG_PER_DATASET, large_blocks_deps);
 	}
+	zfeature_register(SPA_FEATURE_USEROBJ_ACCOUNTING,
+	    "org.zfsonlinux:user_object_accounting", "userobj_accounting",
```

Comment:
Unless there's a compelling reason otherwise, the string after the `:` should be the same as the user-visible name (i.e. `userobj_accounting` and `user_object_accounting` should be the same string).